content/v1.11/getting-started/introduction.md

```diff
@@ -367,95 +322,94 @@ spec:
+    kind: MyComputeResource
   claimNames:
-    kind: computeClaim
+    kind: ComputeClaim
   # Removed for brevity 
 ```
 
 ## Claims
 _Claims_ are the primary way developers interact with Crossplane. 
 
-_Claims_ access the custom APIs defined by the platform team in a _Custom
+_Claims_ access the custom APIs defined by the platform team in a _Composite
 Resource Definition_.
 
-_Claims_ look like _Custom Resources_, but they're namespace scoped,
-while _Custom Resources_ are cluster scoped. 
+_Claims_ look like _Composite Resources_, but they're namespace scoped,
+while _Composite Resources_ are cluster scoped. 
 
 {{< hint "note" >}}
 **Why does namespace scope matter?**  
 Having namespace scoped _Claims_ allows multiple teams, using unique namespaces,
 to create the same types of resources, independent of each other. The compute
 resources of team-A are unique to the compute resources of team-B.
 
-Directly creating _Custom Resources_ requires cluster-wide permissions,
+Directly creating _Composite Resources_ requires cluster-wide permissions,
 shared with all teams.   
-_Claims_ create the same set of resources, but on a namespace level.
+_Claims_ create the same set of resources, but from a namespace level.
 {{< /hint >}}
 
 The previous _Composite Resource Definition_ allows the creation of _Claims_
 of the kind  
-{{<hover label="xrdClaim2" line="7" >}}computeClaim{{</hover>}}.  
+{{<hover label="xrdClaim2" line="7" >}}ComputeClaim{{</hover>}}.  
 
 Claims use the same 
 {{< hover label="xrdClaim2" line="3" >}}apiVersion{{< /hover >}}
 defined in _Composite Resource Definition_ and also used by 
 _Composite Resources_.
 ```yaml {label="xrdClaim2"}
-# Custom Resource Definition (XRD)
+# Composite Resource Definition (XRD)
 spec:
   group: test.example.org
   names:
-    kind: myComputeResource
+    kind: MyComputeResource
   claimNames:
-    kind: computeClaim
+    kind: ComputeClaim
   # Removed for brevity 
 ```
 
 In an example _Claim_ the 
 {{<hover label="claim" line="2">}}apiVersion{{< /hover >}}
 matches the {{<hover label="xrdClaim2" line="3">}}group{{< /hover >}} in the
-_Custom Resource Definition_. 
+_Composite Resource Definition_. 
 
 The _Claim_ {{<hover label="claim" line="3">}}kind{{< /hover >}} matches the
-_Custom Resource Definition_ 
+_Composite Resource Definition_ 
 {{<hover label="xrdClaim2" line="7">}}claimNames.kind{{< /hover >}}.
 
 ```yaml {label="claim"}
 # Claim
 apiVersion: test.example.org/v1alpha1
-kind: computeClaim
+kind: ComputeClaim
 metadata:
-  name: myClaim
-  namespace: devGroup
+  name: my-claim
+  namespace: dev-group
 spec:
   size: "large"
 ```
 
-A _Claim_ can define a {{<hover label="claim" line="6">}}namespace{{</hover >}}.  
-The _Custom Resource Definition_ defines the 
-{{<hover label="claim" line="7">}}spec{{< /hover >}} like a _Custom Resource_.
-
-
-_Claims_ are namespace scoped.
+_Claims_ are namespace scoped. A _Claim_ must specify a {{<hover label="claim"
```

Comment:
They can. I added this because I thought you had to include `namespace: default` in your manifest to create something in the default namespace. I just double checked and that's not the case. If you omit `metadata.namespace` kubectl will create your thing in the default namespace.

_Technically_ this is something `kubectl` is doing for you locally behind the scenes. At the Kubernetes API level you do need to specify a namespace (even if it's `default`). That said, it's probably safe to revert this change.